tasks/gulp/generate-readme.js

```diff
@@ -33,7 +33,8 @@ gulp.task('generate:readme', () => {
     objectData.componentName = path.dirname(file.path).split(path.sep).slice(-1).toString()
     objectData.componentPath = objectData.componentName
     try {
-      let componentData = yaml.safeLoad(fs.readFileSync(`src/components/${objectData.componentName}/${objectData.componentName}.yaml`, 'utf8'), {json: true})
+      let componentPath = path.join(configPath.components, `${objectData.componentName}/${objectData.componentName}.yaml`)
```

Comment:
Suggest letting path.join do it's work here
```
let componentPath = path.join(configPath.components, objectData.componentName, `${objectData.componentName}.yaml`)
```

Reply:
There may be other places where we can use the `path.join` without passing a concatenated string.